tests/baselines/reference/errorWithTruncatedType.errors.txt

```diff
@@ -12,6 +12,7 @@ tests/cases/compiler/errorWithTruncatedType.ts(10,5): error TS2322: Type '{ prop
     
     // String representation of type of 'x' should be truncated in error message
     var s: string = x;
-        ~
+                    ~
 !!! error TS2322: Type '{ propertyWithAnExceedinglyLongName1: string; propertyWithAnExceedinglyLongName2: string; propertyWithAnExceedinglyLongName3: string; propertyWithAnExceedinglyLongName4: string; propertyWithAnExceedinglyLongName5: string; }' is not assignable to type 'string'.
+!!! related TS6230 tests/cases/compiler/errorWithTruncatedType.ts:10:17: Did you mean to access the property 'propertyWithAnExceedinglyLongName1' of { propertyWithAnExceedinglyLongName1: string; propertyWithAnExceedinglyLongName2: string; propertyWithAnExceedinglyLongName3: string; propertyWithAnExceedinglyLongName4: string; propertyWithAnExceedinglyLongName5: string; } instead?

```

Comment:
I'd rather we only issued this on identifiers/property accesses and used that in the error message instead of the type itself. Clearly it can get too unwieldy.

Reply:
Ok, will make the change to only issue if the expression in an entity name (identifier, or property access on an entity name). This will definitely make the error messages easier to understand.